src/parse/ast/bound.ts

```diff
@@ -7,11 +7,16 @@ import type { astToString } from "./utils.ts"
 /**
  * @operator {@link validateBound | bound}
  * @docgenTable
- * @string "[value?][comparitor?]type[comparitor][value]"
- * @comparitors [<,>,<=,>=,==]
- * @example "number<5"
- * @example "2>=number>=5"
- * @example "number===5"
+ * @string "N<S<N", with comparators restricted to < or <=
+ * @param N: number literal
```

Comment:
We shouldn't use param tags here since those indicate function parameters (or maybe generic type parameters but these aren't parameters at all)